packages/frontend/src/app/home/home.component.css

```diff
@@ -20,4 +20,23 @@ limitations under the License.
 	justify-content: center;
 	border-bottom: 1px solid #0075FF;
 	background-color: white;
+}
+
+#presentation-text {
+	margin-top: 10px;
+	width: 100%;
+	text-align: center;
+}
+
+#search-bar {
+	margin-top: 10px;
+	display: flex;
+	width: 100%;
+	align-content: center;
+	justify-content: center;
+}
+
+app-search {
+    width: 100%; /* Take the entire space on mobile devices */
```

Comment:
not to be too nit-picky, but I think we're using different spacing than the CSS above.